.github/renovate.json5

```
@@ -5,8 +5,20 @@
     ":disableDependencyDashboard"
   ],
   "enabledManagers": [
-    // Managers for helm and helm-values. Go dependencies are managed by Dependabot.
-    "helm-values",
+    // Needed to bump the conifgurator version.
+    "regex",
     "helmv3"
   ],
+  "regexManagers": [
+    {
+      "fileMatch": [
+        '(^|/)newrelic-prometheus-agent/Chart\.yaml$'
```

NIT: without the path is enough to test all the charts (or if the chart changes its name). In case there is any other chart that does need Prometheus upgrade too.

```suggestion
        '(^|/)Chart\.yaml$'
```

---

I made this intentionally since the matching string is the `configuratorVersion` and it applies only to this file